### DISCUSSION TITLE
Check Nonce

### DISCUSSION THREAD

**Discussion by octavei** [silius-rs/silius#163] [Q&A]
Usually, we will check the nonce value in the transaction pool, which ensures normal transactions on our chain. Why in op, we don't do this kind of operation ?

**Comment by zsluedem** [comment]
The nonce value should be checked in the simulation step which inside the entrypoint(0.6.0 version)  contract layer(or wallet layer). It is not explicitly checked in the bundler layer.

**Reply by octavei** [reply]
This way, everyone can't submit multiple operations at the same time. Since each op should add 1 to the Nonce on top of the previous op, the chain will definitely not pass when the simulation is executed

**Reply by zsluedem** [reply]
Ok. I see what you are trying to do here. This needs to be additionally taken care of. Are you facing this problem right now?

**Reply by octavei** [reply]
As a user, I want to submit multiple operations at the same time, and  in one TX, they are all able to simulate execution success.
